core/testing/src/main/java/io/github/droidkaigi/confsched2023/testing/robot/TimetableScreenRobot.kt

```diff
@@ -50,12 +53,14 @@ class TimetableScreenRobot @Inject constructor(
 
     fun setupTimetableScreenContent() {
         composeTestRule.setContent {
-            KaigiTheme {
-                TimetableScreen(
-                    onSearchClick = { },
-                    onTimetableItemClick = { },
-                    onBookmarkIconClick = { },
-                )
+            CompositionLocalProvider(LocalClock provides FakeClock) {
```

Comment:
It does not seem like it works well...
How can I overwrite the CompositionLocal in test environment? 🤔

Maybe, since there is no setContent in KaigiAppTest, so the only way is to add like ClockTestRule or something like that?